### ISSUE TITLE
More docs please.

### ISSUE THREAD

**Issue by SurferJeffAtGoogle** [GoogleCloudPlatform/repository-gardener#84]
As I try to review https://github.com/GoogleCloudPlatform/repository-gardener/pull/82, I find it very difficult to understand *the context* in which these scripts will run.  Can you add a `.kokoro/README.md` and answer the questions?
What is the entrypoint for these scripts?
Who runs them?
How often?

**Comment by dzlier-gcp** [comment]
Agreed, it was something I had to track down via both CS and knowledge experts. I plan on tackling this when I write up the process that I followed so others can do so too, but adding a README here could be helpful as well.
